src/plugin.js

```diff
@@ -400,6 +400,79 @@ void main() {
         this.movieScreen.layers.set(2);
         this.scene.add(this.movieScreen);
       }
+    } else if (projection === 'FISHEYE') {
+
+        this.movieGeometry = new SphereGeometry(
+          256,         // radius - sphere´s radius
+          48,          // widthSegments - number of horizontal segments
+          48,          // heightSegments - number of vertical segments
+          0,           // phiStart - specify horizontal starting angle
+          2 * Math.PI, // phiLength - specify horizontal sweep angle size
+          0,           // thetaStart - specify vercial starting angle
+          Math.PI      // thetaLength - specify vertical sweep angle
+        );
+        
+        this.movieMaterial = new MeshBasicMaterial({
```

Comment:
```suggestion
        this.movieMaterial = new THREE.MeshBasicMaterial({
```

Reply:
Glad to hear from you, Brandon! This pull request has been at the back of my mind for some time. It was actually part of the work I did for my (now finished) master´s thesis. I shall provide you with a couple of example assets recorded with the Vivotek FE9391-EV. Let me get back to you shortly!